### PR TITLE
Removed osm and psg logic from Makefile and config

### PR DESCRIPTION
**Pull request by ediakatos**

### CODE REVIEW

**Comment by ediakatos** [comment]
Enhancements on logic for dev day , deletion of db and osm logic

**Review by HSibbenga — COMMENTED** [review]
All changes are fine

**Review by HSibbenga — APPROVED** [review]
All good

**Review by HSibbenga — APPROVED** [review]

**Review by HSibbenga — APPROVED** [review]

**Review by HSibbenga — APPROVED** [review]

**Review by HSibbenga — APPROVED** [review]

**Review by HSibbenga — APPROVED** [review]

File: scripts/Makefile [hunk 1]
```diff
@@ -116,5 +74,5 @@ ${GEOCINT_WORK_DIRECTORY}/geocint/installation/web_dashboard_block_finished: ${G
 
 # ------------------------ MANAGE BLOCK ---------------------------------------
 
-install: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/database_block_finished ${GEOCINT_WORK_DIRECTORY}/geocint/installation/web_dashboard_block_finished ## Target to run Geocint pipeline installation
+install: ${GEOCINT_WORK_DIRECTORY}/geocint/installation/web_dashboard_block_finished ## Target to run Geocint pipeline installation
```
[the remark this answers is not in the document]
Comment (HSibbenga) [reply scripts/Makefile:78]: All good

**Review by HSibbenga — APPROVED** [review]

**Comment by HSibbenga** [comment]
All good

**Review by HSibbenga — APPROVED** [review]

**Review by HSibbenga — APPROVED** [review]

**Review by HSibbenga — APPROVED** [review]
All good

**Review by HSibbenga — APPROVED** [review]
I approve these changes

**Review by HSibbenga — APPROVED** [review]
I approve all these changes

**Pull request closed without merging**